src/fakedmq/neo/request/Consume.d

```diff
@@ -19,6 +19,7 @@ module fakedmq.neo.request.Consume;
 *******************************************************************************/
 
 import dmqproto.node.neo.request.Consume;
+import dmqproto.node.neo.request.core.IRequestResources;
```

Comment:
Or should that be `swarm.common.request.model.IRequestResources` @david-eckardt-sociomantic ?

Reply:
`dmqproto.node.neo.request.core.IRequestResources` is correct. `swarm.common` is legacy swarm, not neo.